msal/src/main/java/com/microsoft/identity/client/MultipleAccountPublicClientApplication.java

```diff
@@ -122,36 +122,17 @@ private void getAccountsInternal(@NonNull final LoadAccountsCallback callback,
         TokenMigrationCallback migrationCallback = new TokenMigrationCallback() {
             @Override
             public void onMigrationFinished(int numberOfAccountsMigrated) {
-                final Handler handler;
-
-                if (null != Looper.myLooper() && Looper.getMainLooper() != Looper.myLooper()) {
-                    handler = new Handler(Looper.myLooper());
-                } else {
-                    handler = new Handler(Looper.getMainLooper());
-                }
-
```

Comment:
We no longer need the handler code?

Reply:
same answer as below. No need to catch exception here, so no handler needed.